src/lib.rs

```diff
@@ -101,12 +101,18 @@ define_if_single_runtime_enabled! {
     pub use ::bson;
 
     mod bson_util;
+
+    #[allow(unused)]
```

Comment:
I don't think these annotations should be on the entire `client`, `db`, and `coll` modules; just put the annotations on the `watch` methods themselves

Reply:
makes sense. fixed